app/src/main/java/fr/free/nrw/commons/filepicker/PickedFiles.java

```diff
@@ -57,15 +57,12 @@ private static File tempImageDirectory(@NonNull Context context) {
      * @param file destination file
      */
     private static void writeToFile(InputStream in, File file) {
-        try {
-            OutputStream out = new FileOutputStream(file);
+        try (OutputStream out = new FileOutputStream(file)) {
             byte[] buf = new byte[1024];
             int len;
             while ((len = in.read(buf)) > 0) {
                 out.write(buf, 0, len);
             }
-            out.close();
-            in.close();
         } catch (Exception e) {
             e.printStackTrace();
         }
```

Comment:
Just catching and printing stack trace isn't going to be helpful since it'll just swallow the exception. Could you remove this catch block and make this method throw IOException. 

I guess the callers are already ready to handle this.

Reply:
this is a good idea 👍 the swallowing of the exception was there prior to this patch, so i was thinking to leave it as it is